app/Bus/Handlers/Commands/Incident/UpdateIncidentCommandHandler.php

```diff
@@ -61,18 +62,20 @@ public function handle(UpdateIncidentCommand $command)
         }
 
         $incident = $command->incident;
-        $incident->update($this->filter($command));
+        $incident->fill($this->filter($command));
 
         // The incident occurred at a different time.
-        if ($command->incident_date) {
-            $incidentDate = $this->dates->create('d/m/Y H:i', $command->incident_date);
-
-            $incident->update([
-                'created_at' => $incidentDate,
-                'updated_at' => $incidentDate,
-            ]);
+        if ($occurredAt = $command->occurredAt) {
+            if ($date = $this->dates->create('Y-m-d H:i', $occurredAt)) {
+                $incident->fill(['occurred_at' => $date]);
+            } else {
```

Comment:
Also, the exception doesn't do anything here, because `create` will throw, so we need to catch.